index.js

```diff
@@ -1,8 +1,6 @@
 /* jshint node: true */
```

Comment:
This file is loaded by the `ng` script, and when you `require('angular-cli')` since it's the `"main"` file in package.